reports/migrated-services/go.mod

```diff
@@ -0,0 +1,18 @@
+module migrated-services
+
+go 1.17
+
+require (
+	github.com/google/go-github v17.0.0+incompatible
+	github.com/ministryofjustice/cloud-platform-environments/pkg/authenticate v0.0.0-20210923104034-4b61cc95504b
```

Comment:
This needs to be github.com/ministryofjustice/cloud-platform-environments v1.1.1.

The package you're importing here is the old sub module screw up (my fault).

Reply:
Good Find. Updated the package path and version